yt/yt/library/containers/cri/cri_executor.cpp

```diff
@@ -704,6 +704,16 @@ class TCriExecutor
             auth->set_registry_token(authConfig.RegistryToken);
         }
     }
+
+    static TRetryChecker GetRetryChecker()
+    {
+        static const auto Result = BIND_NO_PROPAGATE([] (const TError& error) {
+            return IsRetriableError(error) ||
+                   (error.GetCode() == NYT::EErrorCode::Generic &&
+                    error.GetMessage() == "server is not initialized yet");
```

Comment:
Isn't there any GRPC code for this error since error message hard code does not look like reliable?

Reply:
Exactly. This is a bug in containerd. It returns Generic error instead of Unavailable. I've fixed it - see link in commit message.

Reply:
I see. Let's add TODO to rewrite the condition after your fix is merged and deployed.

Reply:
> Broken containerd stay in old distributives forever. Conditions for this TODO will never met. 😒